callgraph/core.py

```diff
@@ -119,19 +127,23 @@ def _MarkExitNodes(self):
                     continue
                 if connection.kind == "nested" or connection.kind == "goto":
                     q.append(self.nodes[connection.dst])
-            
 
-    # Adds to each node information depending on the contents of the code, such as connections
-    # deriving from goto/call commands and whether the node is terminating or not.
+    # Adds to each node information depending on the
+    # contents of the code, such as connections
+    # deriving from goto/call commands and
+    # whether the node is terminating or not.
+

```

Comment:
Please remove this line, I think it's nice when comments are attached to the thing they are explaining.

Reply:
I have removed the line 135.